src/main/java/org/jitsi/jicofo/bridge/SingleBridgeSelectionStrategy.java

```diff
@@ -62,7 +62,7 @@ else if (conferenceBridges.size() != 1)
         }
 
         Bridge bridge = conferenceBridges.keySet().stream().findFirst().get();
-        if (!bridge.isOperational())
+        if (!bridge.isOperational(true /* includeInGracefulShutdown */))
```

Comment:
I find this difficult to understand as to whether or not a bridge in graceful shutdown is being considered here.  For this I'm thinking we could do either:

`if (!bridge.isOperational() || bridge.isInGracefulShutdown())`

or

`if (!bridge.isOperational() && !bridge.isInGracefulShutdown())`

depending on how we want to treat bridges in graceful shutdown here.

Reply:
Done